### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Bigizi**
**Metrics React App**

This project displays the cryptocurrency fetched from the API.
It has the following components:
**index.js:** This displays the cryptocurrency and search bar
**Detail.js:** This displays the details of each coin or cryptocurrency
**Navigation.js:** This displays the navigation icons
There is also a redux folder for configuration and fetching API
I also have the test of each component mentioned above.
N.B: In order to run the test you need to install:  **npm i react-test-renderer**

### CODE REVIEW

File: src/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
```
Comment (rustam-tolipov) [inline src/__tests__/index.test.js:1]: **I looked through your app, it's perfect!**

- [ ] However, when it comes to testing. I only can see snapshots of tests of your components. Students are encouraged to write tests to test the functionality of your app for example fetching and filtering... with a one-word integration test. Kindly add integration tests to your app, Thank you!

File: README.md [hunk 2]
```diff
@@ -150,7 +140,6 @@ $ npm start
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [Issues](https://github.com/Bigizi/Space-Travel-Hub/issues).
```
Comment (rustam-tolipov) [inline README.md:153]: - [ ] Also, in your `ACKNOWLEDGEMENTS` section. I only can see credits for Microverse. But I didn't find the given credits for the creator of the design. Kindly give credit to the designer.

> Requirement: The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 3]
```diff
@@ -40,28 +40,29 @@
...
+## 🚀 Live Demo <a name="live-demo"></a>(Coming soon)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- Loom Walkthrough -->
 ## LOOM PROJECT PRESENTATION <a name="loom-demo"></a>
 
-- [Loom Walkthrough Link](coming soon)
+- [Loom Walkthrough Link](https://www.loom.com/share/55e4214003fe47449424e4eb5f6b408b)
```
Comment (rustam-tolipov) [inline README.md]: - [ ] Great touch on your video description. However, there should be a live demo of your app. Please deploy your app using one of these (Render, Netlify, or GitHub Pages) and add a live demo link in readme.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Bigizi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great design_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (rustam-tolipov) [reply README.md]: - [ ] One more, I think you want to use another title here right? Link to highlight: [link](https://github.com/Bigizi/Metrics-react-app/tree/dev#:~:text=Book%2DStore%2DReact%20Project)

Comment (Bigizi) [reply README.md]: @Rustamxon7 Thank you but I think I will deploy when the project is approved because we deploy from main

Comment (franclobo) [inline README.md]: - [ ] As requested by the previous code reviewer you must deploy your app to approve your project, additionally, it is a mobile version so please 🙏🏽 show the correct size in the video, and kindly show a piece of code that you feel proud of

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Bigizi, 

# ♻️ REQUIRED CHANGES ♻️ 

so-close-jim-halpert

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- Good UI ✔️ 

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 3 above)
Comment (Bigizi) [reply README.md]: @franclobo How I can deploy when the main branch is empty? I thought that the deployment is done from the main branch. Should I record another video?

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**